src/NumericalAlgorithms/Interpolation/InterpolationTargetKerrHorizon.hpp

```diff
@@ -71,7 +71,7 @@ struct KerrHorizon {
   };
   struct ThetaVariesFastest {
     using type = bool;
-    static type default_value() noexcept { return true; }
+    static type suggested_value() noexcept { return true; }
```

Comment:
It's not clear to me that this should be a suggested value — the docs above give a valid use case for setting this to false, and I don't see why we'd want to warn in that use case. I think @markscheel may want to look at this one even though he wasn't assigned to NumericalAlgorithms

Reply:
agreed